Demo/iOS/EditorScreen.swift

```diff
@@ -40,7 +40,7 @@ struct EditorScreen: View {
 private extension EditorScreen {
 
     var editor: some View {
-        RichTextEditor(text: $text, context: context) {
+        RichTextEditor(text: $text, context: context, linkColor: .green) {
```

Comment:
We should probably add a `RichTextEditorStyle` and inject it, so that we can add more styles without having to add more parameters. I can fix that once this is merged.

Reply:
Definitely!